Intake.java

```diff
@@ -0,0 +1,36 @@
+package org.robolancers321.subsystems;
+
+public class Intake extends SubsystemBase{
+
+  public CANSparkMax intakemotor;
+  SparkPIDController IntakePIDController;
+
+  public int kport = 0;
+  public double power = 0;
+  public double kFF = 0;
+
+  public void intake() {

```

Comment:
Constructor should always be upper case (same with the class)

Please run ./gradlew spotlessApply